### ISSUE TITLE
Quests - Greenskin Ch12 - Crushin' Dem All cannot handin quest

### ISSUE THREAD

**Issue by goldo7** [WarEmu/WarBugs#16878]
I found an old bugreport - Quests - Greenskin Ch12 #1931- https://github.com/WarEmu/WarBugs/issues/1931

**Expected behavior and actual behavior:**

Expected: Handin quest to questgiver Grimzag Uruh who has the orange quest completed icon above his head. When talking he shows the quest with the orange icon to the left.

Actual: When clicking the complete button nothing happens. I do notice that only the XP reward is visible. I cannot select an item reward.

**Steps to reproduce the problem:**

/reloadui
logoff/on
abandon q and kill squigs and troll again

same result - cannot handin quest

**Testing Screenshots/Videos/Evidences (always needed):**

![image](https://user-images.githubusercontent.com/65605393/82329739-0f811d80-99e2-11ea-8ab2-da48778d5947.png)

**Comment by Kaelang** [comment]
![image](https://user-images.githubusercontent.com/60117106/82349994-aeae1100-99f2-11ea-89af-876dc58b1b9b.png)

If you can't hand the quest in, I think it's a UI issue. It sounds like your client identifies the rewards, where your 'Complete' button is. 

Please can you disable all of your UI's and try to hand the quest in?

**Comment by goldo7** [comment]
Hello,

I renamed cache, interface and users. So I think I restarted with a clean game?

It does not appear to make a difference.

![image](https://user-images.githubusercontent.com/65605393/82368189-4886b580-9a15-11ea-9525-5fe363d1ee16.png)
![image](https://user-images.githubusercontent.com/65605393/82368283-75d36380-9a15-11ea-82a9-87ff19bba4b9.png)

Abbandoned quest and completed again, no change.

![image](https://user-images.githubusercontent.com/65605393/82369001-a23baf80-9a16-11ea-9ec0-525636575324.png)
